packages/mdc-shape/_functions.scss

```diff
@@ -114,15 +87,26 @@
       @error "Invalid radius: '#{$radius}' is more than 4 values";
     }
 
-    $radius-value: ();
+    $radius-values: ();
 
-    @each $corner in $radius {
-      $radius-value: append($radius-value, mdc-shape-prop-corner-value_($corner));
```

Comment:
I ended up merging the logic of `mdc-shape-prop-corner-value_` into this function after some of the finagling I did led to this function otherwise pretty much not doing anything important in itself besides validating list length.